man/tpm2_quote.1.md

```diff
@@ -0,0 +1,82 @@
+tpm2_quote 1 "SEPTEMBER 2017" tpm2-tools
+==================================================
+
+NAME
+----
+
+tpm2_quote(1) - Provide a quote and signature from the TPM.
+
+SYNOPSIS
+--------
+
+`tpm2_quote` [OPTIONS]
+
+DESCRIPTION
+-----------
+
+tpm2_quote(1) Provide quote and signature for given list of PCRs in given algorithm/banks.
+
+OPTIONS
+-------
+
+  * `-k`, `--akHandle`=_AK\_HANDLE_:
+
+    Handle of existing AK.
+
+  * `-c`, `--akContext`=_AK\_CONTEXT\_FILE_:
+
+    Filename for the existing AK's context.
+
+  * `-P`, `--akPassword`=_AK\_PASSWORD_:
+
+    specifies the password of _AK\_HANDLE_. Passwords should follow the
+    password formatting standards, see section "Password Formatting".
+
+  * `-L`, `--selList`=_PCR\_SELECTION\_LIST_:
+
+    The list of pcr banks and selected PCRs' ids for each bank.
+    _PCR\_SELECTION\_LIST_ values should follow the
+    pcr bank specifiers standards, see section "PCR Bank Specfiers".
+
+  * `-o`, `--outFile`:
+
+    Output file path, recording the two structures output by tpm2_quote function.
+
+  * `-q`, `--qualifyData`:
+
+    Data given as a Hex string to qualify the  quote, optional. This is typically
+    used to add a nonce against replay attacks.
+
+  * `-S`, `--input-session-handle`=_SESSION\_HANDLE_:
+    Optional Input session handle from a policy session for authorization.
+
+[common options](common/options.md)
+
+[common tcti options](common/tcti.md)
+
+[password formatting](common/password.md)
+
+[pcr bank specifiers](common/password.md)
+
+EXAMPLES
+--------
+
+```
+tpm2_quote -k 0x81010002 -P abc123 -g 0x4 -l 16,17,18 -o outFile001
+tpm2_quote -c ak.context -P abc123 -g 0x4 -l 16,17,18 -o outFile001
+tpm2_quote -k 0x81010002 -g 0x4 -l 16,17,18 -o outFile001
+tpm2_quote -c ak.context -g 0x4 -l 16,17,18 -o outFile001
+tpm2_quote -k 0x81010002 -P 123abc -X -L 0x4:16,17,18+0xb:16,17,18 -o outFile001 -q 11aa22bb
```

Comment:
Please remove the -X option that's no longer valid.

Reply:
Done